app/build.gradle

```diff
@@ -177,6 +177,7 @@ dependencies {
     implementation "com.squareup.retrofit2:converter-gson:$retrofitVersion"
     implementation "com.squareup.retrofit2:converter-scalars:$retrofitVersion"
     implementation 'com.squareup.okhttp3:logging-interceptor:3.11.0'
+    implementation project(':twilio-audio-router')
```

Comment:
Given that we will open source this and it will be a pretty standalone thing we may want to nix the `twilio` prefix. It will have `com.twilio` as the package name.